client/db/migrations/1686166769434-CreateBusinesses.ts

```diff
@@ -0,0 +1,21 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class CreateBusinesses1686166769434 implements MigrationInterface {
+  name = 'CreateBusinesses1686166769434';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `CREATE TABLE "businesses" ("id" SERIAL NOT NULL, "created_at" TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(), "updated_at" TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(), "deleted_at" TIMESTAMP WITH TIME ZONE, "profile_id" integer NOT NULL, "name" character varying NOT NULL, "metadata" jsonb NOT NULL, CONSTRAINT "PK_bc1bf63498dd2368ce3dc8686e8" PRIMARY KEY ("id"))`,
```

Comment:
lets remove profile_id as we decided we don't want to have relation to profiles for now